staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -329,7 +333,7 @@ func (d *DiscoveryClient) ServerVersion() (*version.Info, error) {
 
 // OpenAPISchema fetches the open api schema using a rest client and parses the proto.
 func (d *DiscoveryClient) OpenAPISchema() (*openapi_v2.Document, error) {
-	data, err := d.restClient.Get().AbsPath("/swagger-2.0.0.pb-v1").Do().Raw()
+	data, err := d.restClient.Get().AbsPath("/openapi/v2").SetHeader("Accept", mimePb).Do().Raw()
```

Comment:
Cannot change this until 1.11 to maintain one version skew compatibility in kubectl

Reply:
we can do that or we can try new endpoint first and then fallback on the old one. Pro: we have new endpoints in one version and if there is any problem, we would figure it out before completely remove the old endpoint. Con: new clients will have a little more latency to talk to old clusters.

Reply:
I was working on the fallback logic after I sent out the PR. Thanks for pointing that out :) Updated.